lib/fluent/plugin/in_prometheus_monitor.rb

```diff
@@ -95,5 +95,13 @@ def labels(plugin_info)
         type: plugin_info["type"],
       )
     end
+ 
+    def get_gauge(name, docstring)
+      if @registry.exist?(name)
+        @registry.get(name)
+      else
+        @registry.gauge(name, docstring: docstring, labels: @base_labels.keys + [:plugin_id, :plugin_category, :type])
```

Comment:
Not really fond of duplicating the labels `[:plugin_id, :plugin_category, :type]`, could be abstracted in an attribute or a method to align labels here and in `labels` method line `91`. Any feedback?
Same for the 2 other monitoring plugins